src/main/java/com/gascalc/model/GasCalcModel.java

```diff
@@ -13,11 +13,7 @@ public GasCalcModel(String carMake, String carModel, int year) {
 		this.year = year;
 	}
 	
-	
-	public GasCalcModel() {
-		// TODO Auto-generated constructor stub
-	}
-
+	public GasCalcModel() {}
```

Comment:
I removed an unneeded todo within the code. This removes unnecessary whitespace and lowers the size of the executable running, improving efficiency(albeit only a little).